qiita_pet/templates/list_analyses.html

```diff
@@ -2,9 +2,15 @@
 
 {% block head %}
 <script type="text/javascript">
+
+
   $(document).ready(function(){
     {% if analyses %}
       $('#no-analyses').hide()
+      $('#analyses-table').dataTable({
+        "order": [[ 3, "desc" ]]
+      });
+      $('#analyses-table').DataTable();
```

Comment:
I'm confused about this, but isn't this call to DataTable() overriding the settings set in the previous lines? Or do you need these two?